configs/components/ruby-3.2.2.rb

```diff
@@ -223,7 +225,7 @@
     elsif platform.name =~ /osx-12/
       rbconfig_changes["CC"] = 'clang -target arm64-apple-macos12'
     else
-      rbconfig_changes["CC"] = "gcc"
+      rbconfig_changes["CC"] =  'gcc -E'
```

Comment:
I didn't recognize this option. The man page says: 

>  -E  Stop after the preprocessing stage; do not run the compiler proper. 

Is this going to break other platfroms?

Reply:
That might do nothing, I didn't mean to leave that in there. Forgot even what it was trying to do, let me try a build where I remove that and see if it affects anything.

Reply:
Was able to build ruby-3.2.2 just fine without the option, so removed it.